### ISSUE TITLE
Wont open on Mac even after the terminal command. 

### ISSUE THREAD

**Issue by exitramp505** [bisq-network/bisq2#1943]
I updated BISQ 2 and now it won't open. 

I am using a Mac M2 (Sonoma 14.2.1)

https://github.com/bisq-network/bisq2/assets/119704679/79e4db8f-911a-4556-9ed4-a3441b284aa1

**Comment by HenrikJannsen** [comment]
Is there any error message? Can you check if you allow apps which are not from app store?
<img width="481" alt="Screenshot 2024-03-28 at 12 30 33" src="https://github.com/bisq-network/bisq2/assets/116298498/40dd84fc-9706-47de-874e-de94e049fe1d">

**Comment by exitramp505** [comment]
_Is there any error message?_
No, there is no error message. I tried redownloading and I go the same results.
 
_Can you check if you allow apps which are not from app store?_
Yes, this setting had been turned on.

**Comment by HenrikJannsen** [comment]
Hm.... Myself and others can run it on Silicon Apple machines. Can you check the OS settings if there might be another setting which might cause it?

**Comment by ItoroD** [comment]
not sure my problem is related, but I have been trying to run 2.0.2 since last night. The UI won't start up although I can see the bisq app in the application tray and I am seeing notification popup messages from the support chat. All was working fine earlier in the day. I suspect network issues. I get this error kind of multiple times


![image](https://github.com/bisq-network/bisq2/assets/17571526/5b59c071-3739-4cdd-b523-889013638287)

I also see a :    

`03:50:06.736 WARN  [NetworkService.dispatcher-0] b.n.p.n.a.t.h.HashCashTokenService: Message payload not matching proof of work payload. getPayload(message)=`

**Comment by HenrikJannsen** [comment]
Can you try to delete the `tor` directory inside the application directory? If that does not resolve it delete the `network_db` folder inside `db` as well as lock files (shut down bisq before). A restart of the computer might help as well in case a port might be blocked from a previous not terminated process.

@alvasw Any idea? Could it be that when we shut down tor that a lock file stay locked or ports stay open? Or any other reason caused by a non graceful shutdown (I reduced the shutdown timeouts)?

**Comment by ItoroD** [comment]
Can I get specific path to these directories? So I do not delete the wrong thing. As I can see a number of tor dir and its not in application dir directly

**Comment by ItoroD** [comment]
db and tor folder in AppData for me.

I deleted tor dir and network_db in db dir. I also restart machine. Same issue

[bisq.log](https://github.com/bisq-network/bisq2/files/14805535/bisq.log)

**Comment by HenrikJannsen** [comment]
Can you try with a new data directory? Just back up the existing or move it... Can you provide more logs. I can run Bisq 2 without issues, so it seems its a local issue on your side or with the local data

**Comment by ItoroD** [comment]
Backing up the dir folder works. The UI comes up now no errors.

For reference here are the old logs for when I had failure. (Let me know if there is log from another location needed)

[bisq_3.log](https://github.com/bisq-network/bisq2/files/14805826/bisq_3.log)

[bisq_2.log](https://github.com/bisq-network/bisq2/files/14805880/bisq_2.log)
[bisq_1.log](https://github.com/bisq-network/bisq2/files/14805882/bisq_1.log)
[bisq_4.log](https://github.com/bisq-network/bisq2/files/14805883/bisq_4.log)

**Comment by HenrikJannsen** [comment]
So with a new data directory all is ok now? Can you update to 2.0.2 and try with the old data dir again?
You can safely delete all sub dirs in `db` except the 'private` folder. If you delete settings you need to accept again TAC and update settings if you edited some. cache and network_db can be deleted without any problems.

**Comment by ItoroD** [comment]
Yes, with new dir all is OK. 

Alright I will try upgrading to 2.0.2 and revert to old data directory and see

**Comment by HenrikJannsen** [comment]
If you find out that with one particular folder it fails it would be helpful. But I guess its something from past versions and resolved.

**Comment by ItoroD** [comment]
I have upgraded to 2.0.2 

- reverted to my old db folder (the one that was not working)
- deleted sub dir in db folder except private and settings
- backed up old logs so logs starts fresh
- started bisq 2.0.2 application

It refuses to start up. Its taking so long. Basically it is exhibiting the same behavior I had when I was using 2.0.1 with this problematic db dir.

- I switched back to  the working db dir from 2.0.1
- started up 2.0.2
- the app starts up successfully 

probably something about that particular db dir
